api/README.md

```diff
@@ -1,54 +1,50 @@
-# Nixpkgs graph explorer API
+# `nixpkgs-graph-explorer` -  `api`
 
-The REST API for the `nixpkgs-graph-explorer` project, written using
-[FastAPI](https://fastapi.tiangolo.com/).
+An API server to store and serve the data extracted by nixpkgs-graph-explorer core.
 
-## Development
+## Usage
+
+### Load data
 
-To install the application you can run the following from the Nix shell defined
-in the project's [flake.nix](../flake.nix):
+This package comes with a command line utility called `explorer-ingest-graph` to load a Nix graph JSON file that follows [`nixpkgs-graph.schema.json`](../core/nixpkgs-graph.schema.json).
+To extract such JSON, see [the `core` package instructions](../core/README.md).
 
 ```bash
-poetry install
+poetry run python -m explorer.api.ingest --graph-json PATH_TO_FILE
```

Comment:
> I would keep poetry install as a first step, because it must be the first step  Whereas in the current state, people will try this first, and it will fail because they didn't install yet.

Makes sense

> With this change, I would also remove the Set up section below (basically merging the two here, or putting Set up first). I can do the change if you want.

In the end the set up for usage and set up for develop should be different. E.G. once package is on PyPI the install command for usage will be from the artifact, not the source.